spec/unit/graph/simple_graph_spec.rb

```diff
@@ -681,16 +686,20 @@ def graph_to_yaml(graph, which_format)
       end
 
       it "should be able to serialize a graph where the vertices contain backreferences to the graph (#{which_format} format)" do
+        pending('Make SimpleGraph a known to the Puppet Type System')
```

Comment:
Could we fix this test so it's not skipped (I assume by setting `rich_data => false`)? And add a new pending test for the rich data case?